components/footer/Footer.tsx

```diff
@@ -38,6 +40,30 @@ export default function Footer() {
             <FaGithub />
           </SocialButton>
         </Stack>
+        <Box>
+          <NextLink
+            href={
+              'https://github.com/Developer-DAO/academy/issues/new?assignees=&labels=needs+triage%2C+bug&template=bug_report.md&title='
+            }
+            passHref
+          >
+            <Link isExternal textDecoration="underline">
+              Where to send feedback
+            </Link>
+          </NextLink>
+          {'. '}
```

Comment:
Is this space after the full stop intentional?

Reply:
Yeah, that was my lame attempt at spacing! ;-) I'll remove the "period and space" and do it properly!